Update role-based-access-control.md (#2839)
MicrosoftDocs/memdocs: memdocs/intune/fundamentals/role-based-access-control.md

- **Basics**: The assignments name and description.
- **Members**: All users in the listed Azure security groups have permission to manage the users/devices that are listed in Scope (Groups).

> - **Scope (Groups)**: Scope Groups are Azure AD security groups of users or devices or both for which administrators in that role assignment are limited to performing operations on. For example: deployment of a policy or application to a user, or remotely locking a device. All users/devices in these Azure AD security groups can be managed by the users in Members.

Reviewer: ```suggestion
- **Scope (Groups)**: Scope Groups are Azure AD security groups of users or devices or both for which administrators in that role assignment are limited to performing operations on. For example deployment of a policy or application to a user or remotely locking a device. All users and devices in these Azure AD security groups can be managed by the users in Members.
```

- **[Scope (Tags)](scope-tags.md)**: Users in Members can see the resources that have the same scope tags.

> [!NOTE]

> > Scope Tags are freeform text values that an administrator defines and then adds to a Role Assignment. The scope tag added on a role controls visibility of the role itself. while the scope tag added in role assignment limits the visibility of Intune objects (policies, apps, and so on) or devices to only administrators in that role assignment because the role assignment contains one or more matching scope tags.

Reviewer: ```suggestion
> Scope Tags are freeform text values that an administrator defines and then adds to a Role Assignment. The scope tag added on a role controls visibility of the role itself, while the scope tag added in role assignment limits the visibility of Intune objects (such as policies and apps) or devices to only administrators in that role assignment because the role assignment contains one or more matching scope tags.
```

> > Scope Tags are freeform text values that an administrator defines and then adds to a Role Assignment. The scope tag added on a role controls visibility of the role itself. while the scope tag added in role assignment limits the visibility of Intune objects (policies, apps, and so on) or devices to only administrators in that role assignment because the role assignment contains one or more matching scope tags.

Reviewer: ```suggestion
> Scope Tags are freeform text values that an administrator defines and then adds to a Role Assignment. The scope tag added on a role controls visibility of the role itself, while the scope tag added in role assignment limits the visibility of Intune objects (such as policies and apps) or devices to only administrators in that role assignment because the role assignment contains one or more matching scope tags.
```

- **Scope (Groups)**: All users/devices in these Azure security groups can be managed by the users in Members.
- **[Scope (Tags)](scope-tags.md)**: Users in Members can see the resources that have the same scope tags.

> [!NOTE]

> > The scope tags on your roles are not the ones that are used to control what the admins can actually see, but just setting up a scope tag for that specific role.

Reviewer: ```suggestion
> The scope tags on your roles do not control what admins can see, they are only set up for that specific role.
```
